meilisearch/index.py

```diff
@@ -251,7 +252,13 @@ def search(self, query: str, opt_params: Optional[Dict[str, Any]] = None) -> Dic
         query:
             String containing the searched word(s)
         opt_params (optional):
-            Dictionary containing optional query parameters
+            Dictionary containing optional query parameters.
+            Note: The vector parameter is only available in Meilisearch >= v1.3.0, and is experimental
+            Meilisearch v1.3.0. In order to use this feature in Meilisearch v1.3.0 you first need to
```

Comment:
```suggestion
            In order to use this feature in Meilisearch v1.3.0 you first need to
```

Reply:
I may have misread it, but I don't see the point of repeating the version here, no?

Reply:
There may be a better way to say this, let me explain a bit further what I am trying to get across. This feature is only available in 1.3.0+, in addition you have to enable it in 1.3.0 because it is experimental. So the first part will always be true and the second part is only true until it is no longer experimental.

Reply:
> There may be a better way to say this
Probably but I don't have any proposition, either

Reply:
And yes, I understand perfectly what you mean, I find your explanation very clear. 
However, I got the impression that the repetition of the version in this place was unnecessary, but that's just my opinion.
I approve it anyway 😊

Reply:
Would you like me to merge all the PRs for the release preparation, or would you prefer to do it yourself?

Reply:
I'm happy either way, whatever is easier for you.

Reply:
Either way, both suit me. I will do it right away, so it's done 😎